sdk/src/feature_set.rs

```diff
@@ -94,6 +94,10 @@ pub mod rewrite_stake {
     solana_sdk::declare_id!("6ap2eGy7wx5JmsWUmQ5sHwEWrFSDUxSti2k5Hbfv5BZG");
 }
 
+pub mod bpf_just_in_time_compilation {
+    solana_sdk::declare_id!("3bZhti6PMmsdkPxqPT8TPNzB3n9NYbKtnwqEpUDnFuPi");
```

Comment:
Why do we need a feature for this?

Reply:
The idea is that the cli flag is temporary for testing and that we will roll it out using a feature gate later. However, you are right that we don't need the feature right now, but it also does not hurt to have it already.

Reply:
Results should be bit-exact to the current code though, no?

Reply:
I feel like we shouldn't need to feature gate this.  The BPF VM should behave the same with/without JIT, so it's a validator decision to enable/disable.  Initially the `--bpf-jit` argument would be required to enable, then once we're comfortable with it then that turns into a `--no-bpf-jit` argument to disable it